### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by nkreer** [derf/travelynx#37]
Overall, there are a couple of accessibility issues with travelynx that should be rather trivial to fix. This issue was created during my initial exploration of the service using Android's Talkback screen reader and serves as a collection of my findings:

* Buttons in the main navbar (the purple one) are not labelled accurately and their code identifiers are read aloud, e.g. invert_colors (including the underscore) when it should be just "Invert Colors". This can be circumvented by setting an additional aria-label attribute with a correct, localized description of the functionality. This extends to buttons with symbols where things like "comment Kommentar" are spoken aloud. To hide certain elements from a screenreader but still display them visually, set the aria-hidden="true" attribute.
* Some buttons/links like "ZUGLAUF" (likely due to the capitalization) or double-digit carriage numbers are all spelled individually, e.g. "Z U G ..." or "one one" instead of "eleven" 
* Some buttons/links are redundant, e.g. one icon for sharing and a button to "Teilen" right next to it. See problems above. These should be combined to be a single element and the redundant information hidden away from the user.
* Dropdowns (e.g. the station selector) are entirely inaccessible and thus it is mandatory to always write out the entire name of a station
* The error text of textboxes is spoken immediately when they come in focus, even when nothing was entered yet

Thanks for the service! Looking forward to use it in the future! :)

**Comment by derf** [comment]
Indeed, that's an important issue. Thanks for the pointers!
